### ISSUE TITLE
Consume CRDC-H model from new location

### ISSUE THREAD

**Issue by joeflack4** [cancerDHC/ccdh-terminology-service#112]
It's in PyPI now https://pypi.org/project/crdch-model

**Comment by gaurav** [comment]
Hey, we could include the schema file in YAML in that package if that would help!

**Comment by joeflack4** [comment]
@gaurav Yes, that would be helpful! Actually I had thought it was already included w/in the PyPi packages data files, but I hadn't checked. If you do this and you can remember to update me, let me know.

**Comment by gaurav** [comment]
I've opened an issue for this at https://github.com/cancerDHC/ccdhmodel/issues/162
